From wfp5p@virginia.edu  Thu Mar 19 11:49:21 2009
From: Bill Pemberton <wfp5p@virginia.edu>
Date: Thu, 19 Mar 2009 14:22:44 -0400
Subject: Staging: comedi: Remove 2.4 irqreturn_t compatibility in comedi/interrupt.h
To: greg@kroah.com
Message-ID: <20090319182244.5432.86020.stgit@viridian.itc.Virginia.EDU>

From: Bill Pemberton <wfp5p@virginia.edu>

This allows the comedi driver to build after commit bedd3..058bb
is applied.

Signed-off-by: Bill Pemberton <wfp5p@virginia.edu>
Reported-by: Sachin Sant <sachinp@in.ibm.com>
Reported-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/interrupt.h |    7 -------
 1 file changed, 7 deletions(-)

--- a/drivers/staging/comedi/interrupt.h
+++ b/drivers/staging/comedi/interrupt.h
@@ -21,13 +21,6 @@
 
 #include <linux/interrupt.h>
 
-#ifndef IRQ_NONE
-typedef void irqreturn_t;
-#define IRQ_NONE
-#define IRQ_HANDLED
-#define IRQ_RETVAL(x) (void)(x)
-#endif
-
 #ifndef IRQF_DISABLED
 #define IRQF_DISABLED           SA_INTERRUPT
 #define IRQF_SAMPLE_RANDOM      SA_SAMPLE_RANDOM
